src/components/FolderInfo.js

```diff
@@ -0,0 +1,32 @@
+import avatar from "../assets/avatar.svg";
+import "../css/FolderInfo.css";
+
+function UserInfo({ folderData }) {
+  return (
+    <div className="profile-box">
+      <img
+        className="user-image"
+        src={folderData ? folderData.folder.owner.profileImageSource : avatar}
+        alt="avatar"
+      />
+      <span className="user-name">
+        @{folderData ? folderData.folder.owner.name : null}
+      </span>
+    </div>
+  );
+}
+
+function FolderInfo({ folderData }) {
+  return (
+    <div className="folder-container">
+      <div className="folder-info">
+        <UserInfo folderData={folderData ? folderData : null} />
+        <span className="folder-name">
+          {folderData ? folderData.folder.name : null}
```

Comment:
```suggestion
          {folderData?? folderData.folder.name }
```
https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing
위처럼 작성하면 똑같은 효과를 볼 거에요:)